Sources/ChangelogProducerCore/ChangelogBuilder.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ChangelogBuilder.swift
+//  ChangelogProducerCore
+//
+//  Created by Antoine van der Lee on 16/01/2020.
```

Comment:
Shouldn't the copyright be included in the files?

Reply:
Good point, I'll add that!